assets/js/components/ClusterDetails/ClusterSettings.test.jsx

```diff
@@ -42,4 +42,80 @@ describe('ClusterDetails ClusterSettings component', () => {
       screen.getByRole('button', { name: 'Select Checks for Execution' })
     ).toBeVisible();
   });
+
+  it('given VMware provider, should render the warning banner', async () => {
+    const group = faker.animal.cat();
+    const catalog = catalogCheckFactory.buildList(2, { group });
+    const clusters = clusterFactory.buildList(1, { provider: 'vmware' });
+
+    const state = {
+      catalog: { loading: false, data: catalog, error: null },
+      clustersList: { clusters },
+    };
+    const { id: clusterID } = clusters[0];
+
+    const [StatefulChecksSettings] = withState(<ClusterSettings />, {
```

Comment:
I don't think we need still to merge with `defaultInitialState`?
Did you try simply with `withState(<ClusterSettings />, state)` ?

Reply:
Yep, I tried that but an error occurs because it's missing some other data required. I can see if I can mock up the minimal set of data required but as it currently is, `state` is not sufficient.

Reply:
If you really need any value from the `defaultInitialState`, let's put it like:
```
    const state = {
      ...defaultInitialState,
      catalog: { loading: false, data: catalog, error: null },
      clustersList: { clusters },
    };
...
const [StatefulChecksSettings] = withState(<ClusterSettings />, state)
```